packages/frontend/lib/index.ts

```diff
@@ -415,6 +443,13 @@ export default class Nango {
             });
         }
 
+        if ('privateKeyId' in credentials || 'issuerId' in credentials || 'privateKey' in credentials || 'api_key' in credentials) {
+            return await this.triggerAuth({
+                authUrl: this.hostBaseUrl + `/auth/jwt/${providerConfigKey}${this.toQueryString(connectionId, connectionConfig as ConnectionConfig)}`,
+                credentials: credentials as BasicApiCredentials
```

Comment:
BasicApiCredentials?

Reply:
I have updated this with the latest [commit](https://github.com/NangoHQ/nango/pull/2840/commits/6c49f1085afcc716d7d9622373faace0827f0e78)